src/controller.cpp

```diff
@@ -361,6 +487,14 @@ void controller::run(int argc, char * argv[]) {
 
 	LOG(level::INFO, "nl_langinfo(CODESET): %s", nl_langinfo(CODESET));
 
+	if ((! using_nonstandard_configs) && (0 != access(url_file.c_str(), R_OK))) {
```

Comment:
Shouldn't this be `F_OK`? We don't want to migrate if the file exists but we don't have read permissions.